lib/Doctrine/DBAL/Driver/Statement.php

```diff
@@ -19,6 +19,8 @@
 
 namespace Doctrine\DBAL\Driver;
 
+use Doctrine\DBAL\DBALException;
```

Comment:
Basically we adopted the PDO interface back then. PDO heavily relies on returning `false` in case of an error and does not throw exceptions under normal circumstances. The problem is that our implementations are not completely respecting this fact and the only way to go IMO is to try to adhere to the contract. All of those thrown exceptions are bugs so to say. Also it's weird to have a contract that says "in case of an error I return `false` but I could also throw an exception every now and then" :D